vms/shift/utils.py

```diff
@@ -36,6 +36,8 @@ def create_event_with_details(event):
     Creates and returns event with passed name and dates
     """
     e1 = Event(name=event[0], start_date=event[1], end_date=event[2])
```

Comment:
Instead of modifying description attribute afterwards won't it be a better practice to pass it during event __init__ itself.